cdm-test/src/test/java/ucar/nc2/util/net/TestHttpRda.java

```diff
@@ -107,7 +127,7 @@ public TestHttpRda(String ds) {
     this.fullUrl = url + ds;
   }
 
-  @Ignore("Failing - HTTP hangs")
+  //@Ignore("Failing - HTTP hangs")
```

Comment:
I will pass; that test was given to me by Caron.
Not sure it is worth improving.
=Dennis

On 4/3/2016 7:42 PM, Ryan May wrote:
>
> In cdm-test/src/test/java/ucar/nc2/util/net/TestHttpRda.java 
> <https://github.com/Unidata/thredds/pull/513#discussion_r58322597>:
>
> > @@ -107,7 +127,7 @@ public TestHttpRda(String ds) {
> >      this.fullUrl = url + ds;
> >    }
> >
> > -  @Ignore("Failing - HTTP hangs")
> > +  //@Ignore("Failing - HTTP hangs")
>
> So this means we're not ignoring this any more? Again, why not just 
> delete the line?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/Unidata/thredds/pull/513/files/21a7496dffa581258ee5b990ffab97b4e76277cb#r58322597>
>